pkg/policyengine/PolicyDispatcher.go

```diff
@@ -50,7 +47,6 @@ const (
 )
```

Comment:
Can you remove also MBG acronyms from the code.

Reply:
Done